dashboard-generator/generator.js

```diff
@@ -134,14 +135,20 @@ window.addEventListener('load', function () {
       _dash.splice(plotIndex, 1);
       updatePostData();
       tr.parentElement.removeChild(tr);
+
+      if ( _dash.length == 0 ) {
```

Comment:
Style in this file is to not have spaces directly within the parentheses (so this would be `if (_dash.length == 0) {`)